README.md

```diff
@@ -7,10 +7,18 @@ Please fork this repository to your own account, either directly edit the README
 Your statements should include:
 
 * The problem you are addressing
+* ——The accomplishment and engagement rate for online courses are really low.
 * The eductional goal that you plan to address
+* ——My goal is to help course providers estabilish more engagable online course, through which the accomplishment can be increased. 
 * Your objectives for the project
+* ——My objective is to find the features that can improve students' engagement with online courses.
```

Comment:
You are right. Actually, I do have more questions about this project and I really need your advice, so I just signed up for your office hour on 9/27. See you then.